Makefile

```diff
@@ -3,6 +3,7 @@
 NAME = "ProductOpener"
 ENV_FILE ?= .env
 MOUNT_POINT ?= /mnt
+DOCKER_LOCAL_DATA ?= /srv/off/docker_data
```

Comment:
?

Reply:
Talking to Christian, this is tha folder where we may put volumes data.
For what I understood, this volume is not a ZFS clone, it will hold data specific to preprod (this is mostly debug and temporary data, or local data).

Reply:
confirmed with @stephanegigandet, as of now there is no important data there.